docs/buildah-from.md

```diff
@@ -67,6 +67,22 @@ Pull the image even if a version of the image is already present.
 
 If an image needs to be pulled from the registry, suppress progress output.
 
+**--registries-conf** *path*
+
+Pathname of the configuration file which specifies which registries should
+be consulted when completing image names which do not include a registry or
+domain portion.  It is not recommended that this option be used, as the default
+behavior of using the system-wide configuration (frequently
```

Comment:
Lets just say default is /etc/containers/registiers.conf.

I have no idea what frequiently...* preferred means as a user.

Reply:
Changing it.